packages/eas-cli/src/credentials/context.ts

```diff
@@ -90,15 +91,18 @@ class CredentialsContext implements Context {
 
   public logOwnerAndProject() {
     if (this.hasProjectContext) {
+      const owner = getProjectAccountName(this.exp, this.user);
       // Figure out if User A is configuring credentials as admin for User B's project
-      const isProxyUser = this.exp.owner && this.exp.owner !== this.user.username;
+      const isProxyUser =
+        this.user.__typename === 'Robot' || (owner && owner !== this.user.username);
```

Comment:
```suggestion
        this.user.__typename === 'Robot' || owner !== this.user.username;
```
I belive `owner` should be `String` at this point.